Document image overriding on Helm (#820)
submariner-io/submariner-website: src/content/operations/deployment/helm/_index.en.md

        --set serviceAccounts.lighthouseCoreDns.create=true
```

#### Overriding Submariner Images

> The examples below demonstrate how to use images a local registry. It's possible to use an online registry as well.

Reviewer: ```suggestion
The examples below demonstrate how to use images from a local registry. It's also possible to use an online registry.
```
